types/hapi-pino/index.d.ts

```diff
@@ -22,22 +23,27 @@ declare module '@hapi/hapi' {
 }
 
 declare namespace HapiPino {
-    type LogLevels = 'trace' | 'debug' | 'info' | 'warn' | 'error';
-
     interface Options {
         logPayload?: boolean;
         logRouteTags?: boolean;
         stream?: NodeJS.WriteStream;
-        prettyPrint?: boolean;
-        levelTags?: { [key in LogLevels]: string };
-        allTags?: LogLevels;
-        serializers?: { [key: string]: (param: any) => void};
+        prettyPrint?: boolean | pino.PrettyOptions;
+        tags?: { [key in pino.Level]?: string };
+        allTags?: pino.Level;
+        serializers?: { [key: string]: pino.SerializerFn };
+        getChildBindings?: (
+            req: Request,
+        ) => {
+            level?: pino.Level | string;
+            serializers?: Options['serializers'];
```

Comment:
Considering moving this into a separate type.

Reply:
Done